### PR TITLE
Affiliate Page Tweaks

### LINKED ISSUE #13: Affiliate page tweaks
http://dev.bythepixel.com/kyrio/affiliates/

- [x] darken the checkout bg button color on hover (use dark blue in first step with icon)
- [x] left align logo to left page margin
- [x] check logo 1x sizing. (Should be exactly half of natural to keep it from looking fuzzy)
- [x] all dark text color should be the same color (i think #262626)there should be no pure black text.
- [x] add cursor change for Next step button
- [x] add some ease out on the input placeholder animation
- [x] remove animation on checkbox only (make that one snap)
- [x] hide the move date field and reveal it only when the “I am moving..” checkbox is checked
- [x] shorten height of footer (top/bottom padding) check design
- [x] check left/right page margins at all browser sizes (zoom with kyle)
- [x] add kyrio page loader (centered in page) then fade page in quickly
- [x] remove current animation on logo section (open to a different treatment that’s simple)
- [x] check logos for 1x in logo section to keep them looking crisp
- [x] large yellow button - try adding a swipe to the color change.. See this as an example http://www.cssplay.co.uk/menu/cssplay-text-swipe-color-change.html

### PR DESCRIPTION
**Pull request by NathanBurkett**
Closes #13 

## How should the reviewer test this?
http://dev.bythepixel.com/kyrio/affiliates/

- [x] Code Review
- [x] Push to staging
- [x] QA

### QA Checklist
- [x] Ensure checkout button darkens on hover
- [x] Left align logo with page content
- [x] Ensure input form elements' placeholder animation has some easing
- [x] Ensure checkboxes don't have animations for check/uncheck
- [x] Date field should reveal/hide when I'm moving checkbox is checked/unchecked
- [x] Shorten height of footer - see design
- [x] See kyrio page loader and page fade in on page load
- [x] See simplified animation (minimal opacity change) when scrolling to media provider logo section
- [x] Ensure large yellow form button has a text swiping animation on hover (ex:http://www.cssplay.co.uk/menu/cssplay-text-swipe-color-change.html)

### CODE REVIEW

File: assets/js/components/form.js [hunk 1]
```diff
@@ -39,5 +43,71 @@
          }
      });
  }
+
+ const formRevealOnFill = () => {
```
Comment (NathanBurkett) [inline assets/js/components/form.js:62]: Function needs docblock

**Comment by NathanBurkett** [comment]
:+1:

**Comment by CrystalPW** [comment]
QA verified:

Desktop:  OSX Chrome, Firefox  IE 10 (Win 8), 11 and Edge (Win 10)
Tablet: IPad 2
Phone: iPhone 6S

**Comment by wreyniers** [comment]
@CrystalPW - we should also test Internet Explorer 10, 11 and Edge
